files/classes/comment.py

```diff
@@ -248,21 +237,21 @@ def json_raw(self):
 			'flags': flags,
 			}
 
-		if self.ban_reason:
-			data["ban_reason"]=self.ban_reason
-
 		return data
 
 	def award_count(self, kind):
 		if not FEATURES['AWARDS']: return 0
 		return len([x for x in self.awards if x.kind == kind])
 
+	def is_visible(self):
```

Comment:
I suggest removing this function. You can't use it in queries and therefore shouldn't use it in application-side filtering either.

Reply:
Whoops, yeah, I put that in because I was planning to use it and then didn't use it.